gradle-module/src/main/java/org/example/Main.java

```diff
@@ -5,21 +5,23 @@ public class Main {
     public static final String CONSTANT = "constant";
 
     public static void main(String[] args) {
+        String s = "123123";
         System.out.println("Hello world!");
     }
 
     /*
 
     asda
-
-    asd
```

Comment:
[Severity: High] The method 'test' throws a RuntimeException without any handling or context.
 Consider implementing proper exception handling or providing a meaningful message when throwing the exception.
 Rationale: Unhandled exceptions can lead to application crashes and poor user experience. Providing context helps in debugging and maintaining the code.
 Reference: The Java documentation on exceptions emphasizes the importance of handling exceptions gracefully.